go.mod

```diff
@@ -1,14 +1,13 @@
 module github.com/GoogleCloudPlatform/terraform-google-conversion
 
 require (
-	cloud.google.com/go/bigtable v1.0.0
+	cloud.google.com/go/bigtable v1.1.0
 	github.com/apparentlymart/go-cidr v1.0.1
 	github.com/hashicorp/errwrap v1.0.0
-	github.com/hashicorp/terraform v0.12.7 // indirect
-	github.com/hashicorp/terraform-plugin-sdk v1.0.0
-	github.com/terraform-providers/terraform-provider-google v0.0.0-20190926053353-6d9ad351701b
+	github.com/hashicorp/terraform-plugin-sdk v1.4.0
+	github.com/terraform-providers/terraform-provider-google v1.20.1-0.20200227034707-724e0e57d885
```

Comment:
why such a low release?

Reply:
It's being updated from `github.com/terraform-providers/terraform-provider-google v0.0.0` and only has the `version.go` file in the `vendor/terraform-providers/terraform-google-provider` directory.

Is there a reason we need this dependency at all?

Reply:
Good catch. It seems only used in one place at [google/config.go](https://github.com/GoogleCloudPlatform/terraform-google-conversion/blob/master/google/config.go#L273). Any recommendation if we can replace that with something else so we can remove one dependency?

Reply:
Hm, it looks like it is used to create the user agent string. We probably want to change it to use `terraform-conversion` or `terraform-conversion/{version}` instead. That change will need to happen in `config.go.erb` in magic modules and might be tricky because I'm not sure we have a way to identify that we are generating it for the conversion library specifically.

Bottom line, we should get rid of that dependency eventually, but currently it shouldn't break anything